src/homeworks/test_32/index.ts

```diff
@@ -8,3 +10,5 @@
  * @argument data - { name: string; age: number }[]
  * @returns data - { name: string; age: number, position: number }[]
  */
+export const getDataWithPosition = (data: Data[]): getDataAndPos[] =>
```

Comment:
Proviamo a chiamare `data`, `dataList` perche' e' una lista di dati.
```suggestion
export const getDataWithPosition = (dataList: Data[]): getDataAndPos[] =>
```